src/modules/shlvl.rs

```diff
@@ -0,0 +1,53 @@
+use std::env;
+
+use super::{Context, Module};
+
+use crate::config::RootModuleConfig;
+use crate::configs::shlvl::ShLvlConfig;
+use crate::formatter::StringFormatter;
+
+const SHLVL_ENV_VAR: &str = "SHLVL";
+
+pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
+    let shlvl = get_shlvl_value()?;
+
+    let mut module = context.new_module("shlvl");
+    let config: ShLvlConfig = ShLvlConfig::try_load(module.config);
+
+    if config.disabled || shlvl < config.threshold {
```

Comment:
This is a bit of a nitpick but the description says that this is only displayed if the level is greater than the threshold but this will display if the level is greater than or equal to the threshold. Should this be

```suggestion
    if config.disabled || shlvl <= config.threshold {
```
Or maybe the description should be changed.

Reply:
@andytom thanks! i updated - made it a little more clear to reduce ambiguity